custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+name: Litter Box
+products:
+  - id: ebf44398a24b7b6be3vnml
```

Comment:
This appears to be a device id (unique to your device) rather than a product id (shared among all Doel Ti+ devices). Normally a product id should be alphanumeric for the entire length, this one is hex except the last 4 characters, which is how device ids are. I may be wrong about this because there is a small chance that an alphanumeric id just happens to be valid hex, but please double check.